src/token/Token.sol

```diff
@@ -409,6 +424,21 @@ contract Token is IToken, VersionedContract, UUPS, Ownable, ReentrancyGuard, ERC
         return super.owner();
     }
 
+    /// @notice Update minters
+    /// @param _minters Array of structs containing address status as a minter
+    function updateMinters(MinterParams[] calldata _minters) external onlyOwner {
+        // Update each minter
+        for (uint256 i; i < _minters.length; ++i) {
+            // Skip if the minter is already set to the correct value
+            if (minter[_minters[i].minter] == _minters[i].authorized) continue;
+
+            // Update the minter
+            minter[_minters[i].minter] = _minters[i].authorized;
+        }
+
+        emit MintersUpdated(_minters);
```

Comment:
Discussed with james and now that I've read the code and had context, it may make more sense to make this singular and then line 433 makes more sense.

Reply:
Event proposal: `emit MinterUpdated(address minter, bool allowed)`